app/controllers/api/v1/media_controller.rb

```diff
@@ -20,7 +20,7 @@ def show
   end
 
   def update
-    @media_attachment.update!(media_attachment_params)
+    @media_attachment.update!(updateable_media_attachment_params)
     render json: @media_attachment, serializer: REST::MediaAttachmentSerializer, status: status_code_for_media_attachment
```

Comment:
This allows modifying a media attachment's description, focus point and thumbnail without having an edit record or sending the changes through the federation.

Reply:
I have an idea for how to solve this...

Reply:
Oh?

Reply:
I didn't have time to do it yesterday but basically we can run every media attachment update (if it's already attached) through the status update routine. This may create some "noise" if there's a lot of separate updates but it will take care of the problem. Bonus points if there's some way to queue the routine in a debouncing fashion to avoid lots of small updates. Maybe it could even debounce with the "proper" status update if we say that distributing the state is OK if it's 5 minutes after the fact.